gulpfile.js

```diff
@@ -64,10 +59,17 @@ gulp.task('set-environment-config', function (done) {
       });
     }
   });
-});
+};
+gulp.task('set-environment-config', set_environment_config);
+gulp.task('set-env', set_environment_config); // alias
 
+/**
+ * Forcing the environment to be "test".
+ * @param done
+ * @returns TRUE
+ */
 gulp.task('use-test-environment', function (done) {
```

Comment:
This task is imho unnecessary now we have 'test' as the default environment

Reply:
As long i dont have a logic error somewhere, default is still the "production" environment.
If no param ("--production" or "--test") is defined, variable _isProduction_ becomes _true_